user_management/ui/exceptions.py

```diff
@@ -0,0 +1,7 @@
+from django.http import Http404
+from django.utils.translation import ugettext_lazy as _
+
+
+class InvalidExpiredToken(Http404):
+    """Exception to confirm an account."""
+    message = _('Invalid or expired token.')
```

Comment:
As I understand it, Kevin's logic was that the `400` response is actually used and displayed by a frontend, whereas this error isn't handled by anything so we want the browser to show up a `404`.

Technically, I guess it's also more correct - the request is fine, it's the URL that's wrong, since that's where the token is delivered.  That's, like, maximal pedant, though.